DBZ-848 Removed explicit primary node change handling. It is handled by mongoDb driver internally.

tautautau opened the pull request:
Removed explicit primary node change handling. It is handled by mongoDb driver internally. Explicit primary node change event handling was causing issue that no mongo events were read after primary change. This issue only happened when rs.stepDown(<time s>) was used on mongo node.

gunnarmorling commented:
Hey @tautautau, thanks a lot! Could you please alter the commit message so it begins with "DBZ-848" and also use an email as commit author that's linked to your GH account? [That way](https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/#you-havent-added-your-local-git-commit-email-to-your-profile) it will properly show up in the commit stats. Thanks!

tautautau commented:
> Hey @tautautau, thanks a lot! Could you please alter the commit message so it begins with "DBZ-848" and also use an email as commit author that's linked to your GH account? [That way](https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/#you-havent-added-your-local-git-commit-email-to-your-profile) it will properly show up in the commit stats. Thanks!

Hi. I've changed the name and added the mail to the github account. I hope it will work! :)

gunnarmorling commented:
Rebased and applied. Thanks a lot, @tautautau!
